crates/pop-parachains/templates/container/Cargo.templ

```diff
@@ -32,415 +32,8 @@ type_complexity = { level = "allow", priority = 2 }
 unsafe-code = { level = "deny", priority = 1 }
 
 [workspace.dependencies]
-
 container-chain-template-^^template^^-runtime = { path = "runtime", default-features = false }
```

Comment:
Just a question: Does it make sense to keep the `Cargo.templ`, or is simply injecting this line sufficient?

Reply:
It provides some lints for clippy and the workspace,  the package author and repo as well as the members of the workspace.
But This is all something that we could source.

Reply:
I have logged this issue https://github.com/r0gue-io/pop-cli/issues/313 so  we can come back to it and remove any dependency with the askama Cargo template.